### PR TITLE
feat(TCOMP-1983): fix module inclusion in dependencies.txt when build is java9+

### PR DESCRIPTION
**Pull request by undx**
https://jira.talendforge.org/browse/TCOMP-1983

### CODE REVIEW

File: main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/mvn/Mvn.java [hunk 1]
```diff
@@ -81,11 +81,16 @@ public static String locationToMvn(final String location) {
...
         try (BufferedReader reader = new BufferedReader(new InputStreamReader(deps))) {
             // this logic excludes  (optional)  intentionally
-            return reader.lines().map(String::trim).filter(s -> !s.isEmpty() || s.split(":").length < 4)
+            return reader.lines()
+                    .map(s -> s.replaceAll(" -- module.*", "")) // remove java9+ module output
+                    .map(String::trim)
+                    .filter(s -> !s.isEmpty() || s.split(":").length < 4)
                     .filter(s -> !s.endsWith(":test"))
                     .filter(s -> (acceptProvided && s.endsWith(":provided"))
                             || (!acceptProvided && (s.endsWith("compile") || s.endsWith("runtime"))))
```
Comment (ozhelezniak-talend) [inline main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/mvn/Mvn.java:90]: When we accept provided dependency we don't accept compile and runtime?
Is it expected?
Maybe: (!acceptProvided && (s.endsWith("compile") || s.endsWith("runtime") =>(s.endsWith("compile") || s.endsWith("runtime")

**Review by ozhelezniak-talend — COMMENTED** [review]
Put a comment.

File: main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/mvn/Mvn.java (hunk 1 above)
Comment (undx) [reply main/plugins/org.talend.sdk.component.studio-integration/src/main/java/org/talend/sdk/component/studio/mvn/Mvn.java:90]: Yes, it's expected.

**Review by ozhelezniak-talend — APPROVED** [review]
LGTM

**Pull request merged**